app/services/key_rotator/hmac_fingerprinter.rb

```diff
@@ -14,7 +14,7 @@ def rotate(user:, pii_attributes: nil, profile: nil)
 
     def rotate_email_fingerprint(user)
       ee = EncryptedAttribute.new_from_decrypted(user.email)
-      user.update_columns(email_fingerprint: ee.fingerprint)
+      UpdateUser.new(user: user, attributes: { email_fingerprint: ee.fingerprint }).call
```

Comment:
Hey @monfresh when you say "pick one for consistency", what exactly are you referring to? Totally down with using `update_columns` in the key rotator classes, but assume we'd want to stick with `update!` elsewhere?